tests/acceptance/route-tree-test.js

```diff
@@ -129,13 +143,16 @@ module('Route Tree Tab', function(hooks) {
       "PostRoute",
       "application",
       "application",
+      "comments",
```

Comment:
Do we need `"comments"` twice here?

Reply:
@rwwagner90 I double checked and the test seems to want "comments" listed twice similar to "application" and "post".